maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/JGitScmProvider.java

```diff
@@ -54,6 +58,12 @@
 public class JGitScmProvider
     extends AbstractGitScmProvider
 {
+    @Inject
+    public JGitScmProvider( Prompter prompter )
+    {
+        CredentialsProvider.setDefault( new PlexusInteractivityCredentialsProvider( true, prompter ) );
```

Comment:
The `true` looks wrong because you assume that you always run interactive which might not always be the case.

Reply:
https://github.com/apache/maven-scm/pull/153#discussion_r922819246